pygmt/src/info.py

```diff
@@ -31,7 +31,8 @@ def info(table, **kwargs):
 
     Parameters
     ----------
-    table : str or np.ndarray or pandas.DataFrame or xarray.Dataset
+    table : str or numpy.ndarray or pandas.DataFrame or xarray.Dataset or
+            geopandas.GeoDataFrame
```

Comment:
Anyone know how to make a long list of types render correctly in numpydoc? I tried backslashes `\` and putting curly braces around `{'str', 'numpy.ndarray'}` but couldn't figure it out:

![image](https://user-images.githubusercontent.com/23487320/110056169-2d7a0c00-7dc3-11eb-956f-9c91532e6d2d.png)

Reply:
I don't have a solution for this, but I'm wondering can we just say *array-like*, *path-like*, and *file-like*.

Reply:
I'll be handling this in a separate PR at #1186 (while standardizing other modules like plot/plot3d/rose/contour/etc to have the same formatting).

Reply:
The table-like documentation standardization at #1186 should be close to being finalized. Once that PR is merged, I'll update this geopandas integration PR to be like this:

```suggestion
    table : str or {table-like}
```

Reply:
Ok, done in bf72a622cbb041ecefed60182f4ac4b7f5d0c2ca :tada: